### ISSUE TITLE
Heading failures with NVDA and PDF in Chrome and Edge

### ISSUE THREAD

**Issue by brennanyoung** [nvaccess/nvda#14218]
Powermapper reports failures reporting headings and their levels with the combination of NVDA+Chrome+PDF and NVDA+Edge+PDF

https://www.powermapper.com/tests/screen-readers/pdf/pdf-with-headings/

**I suspect that this is a chromium, rather than an NVDA bug**, because NVDA handles this correctly in Firefox (since version 91) and powermapper reports success with NVDA, IE11 and the Acrobat Reader browser plugin. 

This issue was created to track and confirm any fix on the NVDA side when there has been progress with chromium.

The relevant chromium bug seems to be https://bugs.chromium.org/p/chromium/issues/detail?id=1108608&q=component%3AInternals%3EPlugins%3EPDF%3EAccessibility%20headings&can=2

### Steps to reproduce:
https://www.w3.org/TR/WCAG20-TECHS/PDF9.html

### Actual behavior:
NVDA 2022.2 | Chrome 105
NVDA 2022.2 | Edge 105
Only finds one heading, reads wrong heading level. | "Heading level 2, heading 1."

### Expected behavior:
should announce something like "Heading level 1, heading 1. Heading level 2, heading 1.2"
Works correctly with NVDA+IE11+Adobe Acrobat plugin, and NVDA+Firefox

#### NVDA installed/portable/running from source:
Installed 

#### NVDA version:
NVDA 2022.2

#### Windows version:
Windows 10

#### Name and version of other software in use when reproducing the issue:
NVDA 2022.2 | Chrome 105
NVDA 2022.2 | Edge 105
